index.html

```diff
@@ -5,11 +5,61 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Portfolio</title>
+    <link href="https://fonts.cdnfonts.com/css/poppins" rel="stylesheet" />
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <header class="main-header">
-      <a href=""><logo>Noel Bryant</logo></a>
-    </header>
+    <div class="headline-container">
+      <header>
+        <nav class="navbar">
+          <!-- <div class="toolbar"> -->
+          <div class="logo">
+            <a href="">
```

Comment:
- [ ] Please do not leave the `href=" "` blank. You can link it to the homepage. This is for 3 main reasons
  -  Convention
  - Semantics &
  - User experience. You can read more about it [here](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page)

Reply:
- [OPTIONAL]  I think you should not include your logo in the nav tag. The `nav` tag should include the site's navigation menu. Both the logo & the nav however should be included in the `header` tag. 😄